src/service/Fetch.ts

```diff
@@ -1,63 +1,105 @@
+import LAMP from '../index'
```

Comment:
I thought cyclic imports were not permitted - does this compile on older TS versions? (I suppose we don't need to worry about that technically though.)